lib/puppet/provider/package/chocolatey.rb

```diff
@@ -178,7 +178,7 @@ def self.instances
           else
             values = line.split(' ')
           end
-          packages << new({ :name => values[0], :ensure => values[1], :provider => self.name })
+          packages << new({ :name => values[0].downcase, :ensure => values[1], :provider => self.name })
```

Comment:
Oh, I`m sorry!

I was using version 0.5.0, trying to install nssm with this module:

```
class nssm (
	$version = latest,
	$source  = 'SEE NOTES',
) inherits nssm::params {

	package{'nssm':
		ensure   => $version,
		source   => $source,
	}

}
```

where the source was a miror of nuget central. The package is called "NSSM" and puppet returned 2 again and again. After this change it started to work.

I already know it is being done in the caller, but I do not know if this method is overriding an API method that can be called in other place.

Sadly, I cannot reproduce it right now; my whole environment is failing right now with other problem.